cluster/gce/list-resources.sh

```diff
@@ -49,20 +49,20 @@ function gcloud-list() {
   local attempt=1
   local result=""
   while true; do
-    if result=$(gcloud ${group} ${resource} list --project=${PROJECT} ${filter:+--filter="$filter"} ${@:4}); then
-      if [[ ! -z "${GREP_REGEX}" ]]; then
+    if result=$(gcloud "${group}" "${resource}" list --project="${PROJECT}" "${filter:+--filter="$filter"}" "${@:4}"); then
+      if [[ -n "${GREP_REGEX}" ]]; then
         result=$(echo "${result}" | grep "${GREP_REGEX}" || true)
```

Comment:
should it be grep -E?

Reply:
/assign @yujuhong , what do you think?

Reply:
`grep` and `grep -E` supports different syntax. I'd suggest not change this in a cleanup PR.